cm-boy/CmAlgo.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+from CmBark import CmBark
+
+
+class CmAlgo:
```

Comment:
Mark private functions with _ (all functions except for adjust_price & parameter_for_card)
Sort functions according to accessibility.

Reply:
798f8583f89f03c3d4123908194bee5dc129db7a